.github/actions/e2e/action.yml

```diff
@@ -0,0 +1,95 @@
+name: e2e-parallel
+description: "Trigger e2e tests and create reports"
+inputs:
+  apiUrl:
+    description: "API URL from PR or staging"
+    required: true
+  baseUrl:
+    description: "FE URL from PR or staging"
+    required: true
+  commitInfoMessage:
+    description: "Info displayed in commit"
+    required: false
+  cypressGrepTags:
+    description: "Passed tags that should be executed"
+    required: true
+  split:
+    description: "E2E tests containers number"
+    required: true
+  splitIndex:
+    description: "E2E tests container index"
+    required: true
+  install:
+    description: "Triggers cypress installation"
+    required: false
+  browser:
+    description: "Browser which e2e should test"
+    required: false
+  appMountUri:
+    description: "URL for Apps"
+    required: true
+  userName:
+    description: "Name of user which e2e uses"
+    required: true
+  secondUserName:
+    description: "Name of the second user which e2e uses"
+    required: true
+  userPassword:
+    description: "User password for e2e"
+    required: true
+  permissionsUserPassword:
+    description: "User with limited permissions password"
+    required: true
```

Comment:
I think we are always using same user, password etc. Do we need to have it in inputs? I think it always will be secrets.CYPRESS_USER_NAME, etc

Reply:
Yeah, you are right - I will make sure to fix it when new introducing PW in place of cypress. Is that ok?